apollo-audit/apollo-audit-impl/src/main/java/com/ctrip/framework/apollo/audit/controller/ApolloAuditController.java

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright 2023 Apollo Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+package com.ctrip.framework.apollo.audit.controller;
+
+import com.ctrip.framework.apollo.audit.ApolloAuditProperties;
+import com.ctrip.framework.apollo.audit.api.ApolloAuditLogApi;
+import com.ctrip.framework.apollo.audit.dto.ApolloAuditLogDTO;
+import com.ctrip.framework.apollo.audit.dto.ApolloAuditLogDataInfluenceDTO;
+import com.ctrip.framework.apollo.audit.dto.ApolloAuditLogDetailsDTO;
+import java.util.Date;
+import java.util.List;
+import org.springframework.format.annotation.DateTimeFormat;
+import org.springframework.security.access.prepost.PreAuthorize;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+/**
+ * About page: index from 0, default size is 10
+ *
+ * @author luke
+ */
+@RestController
+@RequestMapping("/apollo/audit")
+public class ApolloAuditController {
+
+  private final ApolloAuditLogApi api;
+  private final ApolloAuditProperties properties;
+
+  public ApolloAuditController(ApolloAuditLogApi api, ApolloAuditProperties properties) {
+    this.api = api;
+    this.properties = properties;
+  }
+
+  @GetMapping("/enabled")
+  public String isEnabled() {
+    return "{\"enabled\": "+ properties.isEnabled() +"}";
```

Comment:
return ApolloAuditProperties directly?

```suggestion
    return properties;
```